src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1561,24 +1561,40 @@ class S6Importer final : public IParkImporter
         return justText.data();
     }
 
-    std::vector<rct_object_entry> GetRequiredObjects()
+    template<typename T>
+    static void AddRequiredObjects(std::vector<rct_object_entry>& required, const T& list, size_t internalLimit)
     {
-        std::vector<rct_object_entry> result;
         rct_object_entry nullEntry = {};
```

Comment:
This is used in a number of locations perhaps we should have a constexpr defined for it.

Reply:
There is unfortunately no easy way to constexpr initialize this to have all bytes as 0xFF, it uses union.